client/src/components/Home/Home.css

```diff
@@ -1,4 +1,4 @@
-.logged-in-home {
+.recents-title {
```

Comment:
could you use a more descriptive name for these classes? Does "recents" refer to recent books read? recent activity?